a5py/routines/runmixin.py

```diff
@@ -478,6 +478,9 @@ def getwall_loads(self, weights=True):
             Dropping weights is useful to check how many markers hit a tile
             which tells us how good the statistics are.
 
+        particle_filter : list[int], optional
+            Calculate wall loads only for the particles with the given indices
```

Comment:
Maybe change the name of this argument to `ids` since it is used elsewhere to filter markers. And the change "given indices" to "given IDs." (note dot at the end)

Reply:
Changed to `p_ids` for included particle IDs and `w_indices` for included triangle indices. Left the prepended p and w for clarity to the users.